doc/installation/system/inifile.rst

```diff
@@ -122,6 +122,28 @@ set ``PI_AUDIT_NO_SIGN = True``. If ``PI_AUDIT_NO_SIGN`` is set to ``True``
 audit entries will not be signed and also the signature of audit entries will not be
 verified. Audit entries will appears with *signature* *fail*.
 
+.. _monitoring_modules:
+
+Monitoring parameters
+---------------------
+
+``PI_MONITORING_MODULE`` lets you specify an alternative statistics monitoring module.
+The monitoring module takes care of writing values with timestamps to a store.
+This is used e.g. by the :ref:`eventcounter` and :ref:`taskmodule_simplestats`.
+
+The first available monitoring module is ``privacyidea.lib.monitoringmodults.sqlstats``.
+It accepts the following additional parameters:
+
+``PI_MONITORING_SQL_URI`` can hold an alternative SQL connect string. If not specified the
+normal ``SQL_DATABASE_URI`` is used.
+
+``PI_MONITORING_POOL_SIZE`` (default 20) and ``PI_MONITORING_POOL_RECYCLE`` (default 600) let
+you configure pooling. It used the settings from the above mentioned
```

Comment:
*uses

Reply:
done in 905639b0a9974c9bbbaad243384e8539088fc1a1

Reply:
:+1: